common/modifier_definitions/wc_opinion_modifier_definitions.txt

```diff
@@ -362,6 +362,13 @@ pirate_culture_opinion = {
 	is_hidden = no
 	max_decimals = 0
 }
+drust_opinion = {
```

Comment:
Not localized.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/80980064-3dd6f880-8e39-11ea-8ef0-a1e0736a707b.png)

</details>